### DISCUSSION TITLE
Is there Steam Overlay in Game?

### DISCUSSION THREAD

**Discussion by AlexMerts** [Whisky-App/Whisky#677] [Q&A]
Hey! Has anyone gotten the Steam overlay to work in the game, or do you know if it is possible?

**Comment by VyacheslavFedorovyh** [comment]
I found a way, you need to go to the “bottle configuration” section in Whiskey => enable Meta HUD and Trace Metal
In the terminal "/bin/launchctl setenv MTL_HUD_ENABLED 1"

**Reply by the-toasted-loaf** [reply]
I tried this but now it turns on automatically on launch for all metal apps even when you run the "/bin/launchctl setenv MTL_HUD_ENABLED 0" command.

**Reply by BLOKBUSTR** [reply]
I've tried this a while ago, but it didn't work for me. It sure brings up diagnostic information, but no Steam Overlay.

If I remember correctly, the command to disable the Metal HUD is exactly `/bin/launchctl unsetenv MTL_HUD_ENABLED`, making sure not to write `setenv` instead. I didn't need to use a 0 at the end either. The Metal HUD will still appear in apps/games run by Whisky, so undo the aforementioned options in Bottle Configuration.

I really wish there could be another solution for Steam Overlay. For Lethal Company in particular, it's difficult to effectively host lobbies as it primarily relies on the Steam Overlay to invite friends without providing much alternative.

**Reply by the-toasted-loaf** [reply]
my friend ( on windows) tried inviting me but it just came up with an error for me, but the command worked thanks!

**Reply by BLOKBUSTR** [reply]
Did you try more than once to join the lobby? Because I run into a lot of errors when following others' invites too, but at one point it randomly works.

Alternatively, you can manually join a friend's game by navigating to their profile in the Steam app and finding a "Join Game" button under their game status. It also takes multiple attempts until the game finally lets you in, as far as I've seen.

Is this something Windows users regularly experience too, or is it specific to us Mac users?

**Reply by vafanassieff** [reply]
What is the link between the metal HUD and the steam overlay ?

**Reply by lesha-co** [reply]
> Alternatively, you can manually join a friend's game by navigating to their profile in the Steam app and finding a "Join Game" button under their game status

what's bothering me is that I don't have this button. It only shows "show in steam" button that takes me to the store page (as if I don't own the game. I do and it is installed)

<img width="366" alt="image" src="https://github.com/Whisky-App/Whisky/assets/4255635/7ec99767-3bb7-4320-a7e5-5c4ce128860a">
